lib/gollum/views/compare.rb

```diff
@@ -17,9 +17,9 @@ def after
         @versions[1][0..6]
       end
 
-      def lines
+      def lines(diff = @diff)
```

Comment:
Didn't know you could use an instance variable in a method definition like this! If it works, fine!

Reply:
No problem. The original behavior of `/compare/` is conserved.